src/components/Comments/Comment.js

```diff
@@ -333,7 +334,10 @@ const Comment = ({
               styles.btn,
               styles.iconContainer
             ]}
-            onPress={() => onBlockComponent(comment)}>
+            onPress={() => {
+              console.log(comment);
+              onBlockComponent(comment);
+            }}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While adding a `console.log` statement can be useful for debugging during development, it should be removed in production code to avoid exposing potentially sensitive information. Also, ensure that `onBlockComponent` function handles all edge cases and errors properly.
```diff
-  console.log(comment);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->